occurrences/signals.py

```diff
@@ -43,3 +78,40 @@ def remove_message_id(sender, message, **kwargs):
     # ESP in order to avoid the message being identified as spam by some strict
     # filters.
     message.extra_headers.pop("Message-ID", None)
+
+
+@receiver([m2m_changed], sender=Occurrence.languages.through)
+@transaction.atomic
+def update_event_languages(sender, instance, action, pk_set, **kwargs):
```

Comment:
I think we should have a test for this too

Reply:
Added some tests.